scripts/build_frontend.sh

```diff
@@ -5,8 +5,8 @@ if [ -z "$(ls | grep CONTRIBUTING.md)" ]; then
 else
   echo "Building the frontend"
   cd frontend
```

Comment:
should this be "cd ui"?

Reply:
Ah, yeah it should. I didn't actually know what this script was for.